pages/docs/tooling/tooling-overview.md

```diff
@@ -17,6 +17,7 @@ Here you will find a few tools to:
 - [Analyze the performance of the coupled simulation](tooling-performance-analysis.html) to understand where the runtime comes from.
 - [Compute parameters for the RBF mapping configuration](tooling-rbf-shape.html) to optimize the accuracy and performance of your RBF mapping.
 - [A helper library in Python for performing multiple preCICE runs for convergence and performance studies](https://pypi.org/project/prepesthel/).
+- [A helper library in Python for generating the initial folders and files for a preCICE simulation setup based on a simple topology file](https://github.com/precice/case-generate).
```

Comment:
You could also add the config checker here.

Reply:
see my response [in the other PR](https://github.com/precice/precice.github.io/pull/532#issuecomment-2995252460)